regcore_read/views/es_search.py

```diff
@@ -37,6 +39,10 @@ def search(request, doc_type):
             term['version'] = version
         if regulation:
             term['regulation'] = regulation
+        if is_root:
+            term['is_root'] = is_root
+        if is_subpart:
+            term['is_subpart'] = is_subpart
```

Comment:
I messed around this more and confirmed that passing `"true"` and `"false"` works for both haystack and the elastic rest api. Example:

```
In [2]: SearchQuerySet().models(Document).filter(is_root=False).count()
Out[2]: 9

In [3]: SearchQuerySet().models(Document).filter(is_root=True).count()
GET http://localhost:9200/eregs/modelresult/_search?_source=true [status:200 request:0.002s]
Out[3]: 9

In [4]: SearchQuerySet().models(Document).filter(is_root='false').count()
GET http://localhost:9200/eregs/modelresult/_search?_source=true [status:200 request:0.037s]
Out[4]: 3148

In [5]: SearchQuerySet().models(Document).filter(is_root='true').count()
GET http://localhost:9200/eregs/modelresult/_search?_source=true [status:200 request:0.017s]
Out[5]: 9
```

The counts with string values are correct, and they also work with the regular elastic api. Meanwhile, seems like we should validate that query parameters for these boolean terms are "true" or "false" and not something else--I'll add that.